src/react-apps/applications/ux-editor/src/utils/networking.ts

```diff
@@ -32,9 +32,14 @@ export async function put(
   apiMode: string,
   data: any,
   config?: AxiosRequestConfig,
+  validationTriggerField?: string,
 ): Promise<any> {
   try {
-    const response: AxiosResponse = await axios.put(url + `/${apiMode}`, data, config ? config : null);
+    let putUrl = `${url}/${apiMode}`;
+    if (validationTriggerField) {
+      putUrl += `?validationTriggerField=${validationTriggerField}`;
+    }
```

Comment:
Why can't this just be a part of the url passed in?

Reply:
I guess it could, but since we already pass in the `apiMode` we would have to make the change everywhere this function is used.

Reply:
Could we risk that the path will not be a valid param that needed escaping if we put it in the body this is not a problem. What is the cons against the body solution?

Reply:
Yes it's possible if we look at repeating groups, as these will contain `[]`-characters which can be unsafe in urls. 
If we want to pass it in the body, we have to rewrite the parsing of the service model. as this parsing currently assumes it's only form data that is being sent in the body.

Reply:
@TheTechArch moved the parameter to the header.